selfdrive/car/volkswagen/carcontroller.py

```diff
@@ -39,7 +39,7 @@ def update(self, c, enabled, CS, frame, ext_bus, actuators, visual_alert, left_l
       # torque value. Do that anytime we happen to have 0 torque, or failing that,
       # when exceeding ~1/3 the 360 second timer.
 
-      if c.active and CS.out.vEgo > CS.CP.minSteerSpeed and not (CS.out.standstill or CS.out.steerFaultPermanent or CS.out.steerFaultTemporary):
+      if c.latActive and not CS.out.standstill:
```

Comment:
move the standstill check to controlsd?

Reply:
In another PR I'll replace with the MIN_STEER_SPEED in latcontrol.py, then we can remove lac_log.active as it will be a duplicate of CC.latActive